docs/Fabulous.XamarinForms/testing.md

```diff
@@ -151,6 +152,22 @@ let ``View should generate a valid interface``() =
     stepSizeLabel.Text |> should equal "Step size: 4"
 ```
 
+We can use `tryFindViewElement` to check if a ViewElement is available. We can use it as follows with the view which we declared before:
```

Comment:
Maybe we should be a little bit clearer.

`tryFindViewElement` is not really here to check if a ViewElement exists, but more for quick access to a ViewElement by using its AutomationId.
It's an alternative to the verbose Viewer API:

```fsharp
let contentPage = ContentPageViewer(actualView)
let stackLayout = StackLayoutViewer(contentPage.Content)
let countLabel = LabelViewer(stackLayout.Children.[0])
let timerSwitch = SwitchViewer(StackLayoutViewer(stackLayout.Children.[3]).Children.[1])
let stepSlider = SliderViewer(stackLayout.Children.[4])
let stepSizeLabel = LabelViewer(stackLayout.Children.[5])
```
vs
```fsharp
let stepSizeLabelOpt = tryFindViewElement "stepSizeLabel" actualView
```

So, maybe add 2 sublevels (`####`) presenting (1) how to use the Viewer API's way of accessing things and (2) `tryFindViewElement`?

Checking the existence of a control is only an added nice thing to `tryFindViewElement` that we can mention. :)